nx.json

```diff
@@ -23,17 +15,18 @@
           "stylelint",
           "paths"
         ],
-        "parallel": 1
+        "parallel": 1,
+        "accessToken": "OGI1YjE1ZTMtMDJhMi00MDVlLTkwNWItMWViY2ZjNmNmZTFjfHJlYWQtd3JpdGU="
```

Comment:
Do we need this token? Where ti comes from?

Reply:
It was added with nx-cloud. Will remove it